angular-portfolio/src/app/awards/awards.component.scss

```diff
@@ -0,0 +1,52 @@
+.awards {
+    background-color: #ecf0f1;
+    height: 65vh;
+
+    .flex-container {
+        align-items: center;
+        flex-direction: row;
+        display: flex;
+        position: relative;
```

Comment:
What is the purpose of the relative positioning?

Reply:
Does not make sense in this context. Removed.